tests/jest/unit/BackendClient.spec.js

```diff
@@ -90,3 +91,91 @@ test('BackendClient.getQueryString()', () => {
     a: ['1', '2'],
   })).toEqual('a=1&a=2');
 });
+
```

Comment:
if we end up having to do more mock `fetch` response testing, we might want to either move this somewhere or investigate `jest-mock-fetch`.